gap/Magic.gd

```diff
@@ -56,15 +60,27 @@
 #!
 #! <Mark><A>package</A></Mark>
 #! <Item>
-#!     This is either the name of package, or an <C>IsDirectory</C> object.
-#!     In the former case, &AutoDoc; uses the metadata of the first package
-#!     with that name known to &GAP;. In the latter case, it checks whether
-#!     the given directory contains a <F>PackageInfo.g</F> file, and extracts
-#!     all needed metadata from that. This is for example useful if you have
-#!     multiple versions of the package around and want to make sure the
+#!     The purpose of this parameter is twofold: to determine the package
+#!     directory in which &AutoDoc; will operate, and to find the metadata
+#!     concerning the package being documented. The parameter is either a
+#!     string, an <C>IsDirectory</C> object, or omitted. 
```

Comment:
Then perhaps we should rename it from `package` to `packageOrDirectory`; or `pkgOrDir` ?